### PR TITLE
lib+api: add litd proto files and Lit API class

### PR DESCRIPTION
**Pull request by jamaljsr**
This PR adds the proto files from `litd` and exposes the RPCs via the `Lit` class.

This update relies on https://github.com/lightninglabs/lightning-terminal/pull/487. We'll need to wait for the new release of `litd` before we can merge this PR in order to get the latest version number.

### CODE REVIEW

File: scripts/update_protos.sh [hunk 1]
```diff
@@ -35,3 +37,9 @@ curl ${POOL_URL}/${POOL_RELEASE_TAG}/auctioneerrpc/auctioneer.proto --create-dir
 curl ${POOL_URL}/${POOL_RELEASE_TAG}/auctioneerrpc/hashmail.proto --create-dirs -o protos/pool/${POOL_RELEASE_TAG}/auctioneerrpc/hashmail.proto
 
 curl ${FARADAY_URL}/${FARADAY_RELEASE_TAG}/frdrpc/faraday.proto --create-dirs -o protos/faraday/${FARADAY_RELEASE_TAG}/faraday.proto
+
+# TODO: uncomment these lines once the autopilot protos have been merged into the public LiT repo.
```
Comment (kaloudis) [inline scripts/update_protos.sh:41]: this is dependent on https://github.com/lightninglabs/lightning-terminal/pull/487?

Comment (jamaljsr) [reply scripts/update_protos.sh:41]: yup. i'll need to update this PR after that gets merged and shipped in the next release.

**Comment by jamaljsr** [comment]
Updated to use the official litd `v0.8.6-alpha` tagged release.

**Review by kaloudis — APPROVED** [review]

**Review by kaloudis — APPROVED** [review]
LGTM!

**Pull request merged**